<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QWidget" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainView</string>
  </property>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Note</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="notesTableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>201</width>
     <height>192</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>201</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>10</width>
     <height>0</height>
    </size>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date of creation</string>
    </property>
   </column>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
